### ISSUE TITLE
Loot Won't Sort

### ISSUE THREAD

**Issue by ScorchedSky** [loot/skyrim#162]
Hey so ever since I updated to the latest version of LOOT it hasn't been sorting my load order, or if it does it's hardly noticeable. I did the convert thing for my settings but as far as I can tell it's not sorting my plugins very well if at all. I've tried everything I can think of so either I'm missing something or...I dunno. Anyway I hope someone here can help because up until now I've never had any problems running LOOT. Here's my masterlist or what I understood to be what I need.
[masterlist.docx](https://github.com/loot/skyrim/files/1590278/masterlist.docx)

**Comment by Ortham** [comment]
Masterlists aren't `.docx` files, I don't know how you got that. Please see [How To Report Issues Helpfully](http://loot.github.io/docs/help/How-To-Report-Issues-Helpfully.html).

**Comment by ScorchedSky** [comment]
Oh I converted it so the file would show up here on the post, it wouldn't accept yaml files. I have it as a yaml file on my computer of course.

**Comment by Ortham** [comment]
OK, but please see [How To Report Issues Helpfully](http://loot.github.io/docs/help/How-To-Report-Issues-Helpfully.html).

**Comment by ScorchedSky** [comment]
I'm sorry, I've really never had to do this before so I looked at the report issue page and I'm still not sure what I'm doing wrong reporting the issue. Let me try again. 
[Masterfile.zip](https://github.com/loot/skyrim/files/1594080/Masterfile.zip)

**Comment by Ortham** [comment]
Well, I'm after your debug logs since you said you've only started having an issue after you updated LOOT, so it sounds like an application issue rather than a metadata issue, but I suppose it's ambiguous here.

**Comment by ScorchedSky** [comment]
Ah ok. That makes more sense. Sorry I'm not being very helpful in asking for help. Anyway here are the debug reports: 
[CEFDebugLog.txt](https://github.com/loot/skyrim/files/1594736/CEFDebugLog.txt)
[LOOTDebugLog.txt](https://github.com/loot/skyrim/files/1594737/LOOTDebugLog.txt)

**Comment by Ortham** [comment]
It doesn't look like those logs are for when you tried sorting: the logs get overwritten each time you run LOOT. Can you run LOOT again and try to sort with it, then upload the logs for that?

**Comment by ScorchedSky** [comment]
Alright, I just used LOOT and here are the updated logs:
[CEFDebugLog.txt](https://github.com/loot/skyrim/files/1594826/CEFDebugLog.txt)
[LOOTDebugLog.txt](https://github.com/loot/skyrim/files/1594827/LOOTDebugLog.txt)

**Comment by Ortham** [comment]
Well, it looks like LOOT is working correctly, so if you're not getting the result you expect it's either not a problem or a metadata issue. If you're just expecting some unspecific change to be made, that's not enough to assume something's wrong when it doesn't occur. However, if you know B should be loading after A and LOOT isn't sorting them like that, then that's a specific issue metadata can be added to solve.

**Comment by ScorchedSky** [comment]
The only reason I knew it wasn't sorting was because FNIS always had my FNIS_PCEA2 directly below it. I wondered if it was a metadata issue so I tried clearing it once but that didn't do anything and again I don't know all that much about the finer details of all this sorting stuff. Usually it just works! Ok so what should I try to do next to fix the problem? Or better yet even find out what it is?
